btcjson/walletsvrresults.go

```diff
@@ -173,3 +173,18 @@ type GetBalancesResult struct {
 	Mine      BalanceDetailsResult  `json:"mine"`
 	WatchOnly *BalanceDetailsResult `json:"watchonly"`
 }
+
+// WalletCreateFundedPsbtResult models the data returned from the
+// walletcreatefundedpsbtresult command.
+type WalletCreateFundedPsbtResult struct {
+	Psbt      string  `json:"psbt"`
+	Fee       float64 `json:"fee"`
```

Comment:
This should be `btcutil.Amount`.

Reply:
Fee and Amount are represented as `float64` in every other result. Using `btcutil.Amount` here would introduce an inconsistency.

Reply:
Oops, my bad. The result structs should indeed use `float64`. It's the constructors and `rpcclient` methods that should use `btcutil.Amount`.

Reply:
Coming back to this topic, it's true that most commands in this file use the type `float64` for fees, but we should really compare it with `FundRawTransactionResult`:
https://github.com/btcsuite/btcd/blob/23d149cbfbfa13938dcb64ed62f22250eb6d239d/btcjson/chainsvrresults.go#L710-L715

We can keep it as-is, but should plan a refactoring to convert all amount-like fields to the `btcutil.Amount` type.